src/api/EpiData.ts

```diff
@@ -357,6 +369,7 @@ export function importFluView({
       'num_age_4',
       'num_age_5',
     ],
+    '',
```

Comment:
Ick!  The need to include this empty string is an ugly/unfortunate side effect from the recent addition of the `columnRenamings` argument to `loadDataSet()` in #66 (and because TypeScript doesnt have a "proper" named optional argument syntax).

Does this work?  Its feels a lot less hacky:
```suggestion
    auth, # also remove `auth` from the `userParams` arg a few lines above
```
We should standardize all of the other "`importFoo()`" methods (for the endpoints that use an "authorization token") so they use the `api_key` argument of `loadDataSet()` too.  I can see some rationale for doing that in PR #72, but its big enough already, and it doesnt touch this file, and the argument was introduced in this PR, so i think it belongs here.